### ISSUE TITLE
Keras Models Not Working

### ISSUE THREAD

**Issue by eurus-yu** [davidsandberg/facenet#643]
I have implemented two versions of VGG-16, slim version works fine but keras version is not working. I am using softmax training.For this keras version of VGG-16, the LFW 
 accuracy remains at 56%. I have no idea how this could happen. Please help me, thanks!

**slim version (works):**
https://github.com/eurus-yu/FaceAcceleration/blob/master/source/models/vgg_16.py
**keras version (not working)**
https://github.com/eurus-yu/FaceAcceleration/blob/master/source/models/vgg_16_keras.py

**Comment by eurus-yu** [comment]
I solved it. It's about the BN layers, the mean and variances are not updated...
